fdbserver/storageserver.actor.cpp

```diff
@@ -6609,6 +6904,8 @@ ACTOR Future<Void> storageServerCore(StorageServer* self, StorageServerInterface
 	self->actors.add(serveChangeFeedStreamRequests(self, ssi.changeFeedStream.getFuture()));
 	self->actors.add(serveOverlappingChangeFeedsRequests(self, ssi.overlappingChangeFeeds.getFuture()));
 	self->actors.add(serveChangeFeedPopRequests(self, ssi.changeFeedPop.getFuture()));
+	self->actors.add(serveGetCheckpointRequests(self, ssi.checkpoint.getFuture()));
+	self->actors.add(serveGetFileRequests(self, ssi.getFile.getFuture()));
```

Comment:
This is correct. Just wonder why not putting the logic inside the `loop` below, where we `choose` among different requests, which is probably more readable (by keeping different requests handling at one place) and maybe expose `SlowSSLoopx100` more accurately.

Reply:
Done.